js/event.js

```diff
@@ -57,4 +72,27 @@ $(function() {
 
 function showUnarchiveConfirmation() {
     $('#unarchive-confirmation-wrapper').removeClass('hidden');
-}
\ No newline at end of file
+}
+
+$(function() {
+    // from http://stackoverflow.com/questions/45888/what-is-the-most-efficient-way-to-sort-an-html-selects-options-by-value-while
+    var my_options = $('.facilities select option');
+    var selected = $('.facilities').find('select').val();
+  
```

Comment:
Could you add a comment that briefly describes this section?

Reply:
This function is actually from a previous bit of code that I was working with that is now deleted. I believe it's no longer necessary, that's probably why it doesn't make sense in its context

Reply:
Got it. Thanks for clarifying!